demos/list.html

```diff
@@ -1329,6 +1329,48 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        var deactivateRippleHandler = null;
+        var checkboxListRippleSurfaceClass = 'checkbox-list-ripple-surface';
+
+        function clickHandler(evt, rippleFoundation, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          var isLabel = Array.prototype.some.call(checkbox.labels, function(label) { return label === evt.target; });
+          if(evt.target !== checkbox && !isLabel) {
+            checkbox.checked = !checkbox.checked;
+            li.classList.add(checkboxListRippleSurfaceClass);
+            rippleFoundation.activate();
+
+            deactivateRippleHandler = e => deactivateRipple(e, rippleFoundation, li);
```

Comment:
IE 11 💩  doesn't support arrow functions ☹️   (It also lacks proper `const` or `let` support).

So our uncompiled demo JS needs to use ES5 syntax, sadly.

Reply:
ah...right. I'm just so used to it I didn't even realize.